test/diagnostics_test.cc

```diff
@@ -0,0 +1,43 @@
+
+#include "benchmark/benchmark_api.h"
+#include "../src/check.h"
```

Comment:
please add a comment that this shouldn't be copied by users of the library or something :)